staking/types/validator.go

```diff
@@ -5,6 +5,16 @@ import (
 
 	"github.com/harmony-one/bls/ffi/go/bls"
 	"github.com/harmony-one/harmony/internal/common"
+	"github.com/harmony-one/harmony/internal/ctxerror"
+)
+
+// Define validator staking related const
+const (
+	MaxNameLength            = 70
+	MaxIdentityLength        = 3000
+	MaxWebsiteLength         = 140
+	MaxSecurityContactLength = 140
+	MaxDetailsLength         = 280
 )
```

Comment:
How you picked these magic numbers

Reply:
I copied these numbers, we can always modify it later. This is just a start